### PR TITLE
Update models.py with databases

### PR DESCRIPTION
**Pull request by AnhHoaiNguyen-916**

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -1,11 +1,113 @@
...
+            return
+        # If the token is valid --> the user object corresponding to the user id in the token.
+        return User.query.get(id)
+
+    # Function that returns 1 string every time creating 1 new row (user): 
+    def __repr__(self):
+        return f'<User {self.id}: {self.username}, {self.fullname}>'
+
+# Set up the table Post with different columns in Database:
+class Post(db.Model):
```
Comment (kenneth0810) [inline app/models.py:63]: What is this Post class for?

File: app/models.py [hunk 2]
```diff
@@ -1,11 +1,113 @@
...
+
+# Create all the tables by calling create_all() method of the db object 
+# within the application context to ensure that the app's database connection is available:
+with myapp.app_context():
+    db.create_all()
+
+# This is a callback function used by Flask-Login to load a user from a user ID.
+# The 'id' argument is the user ID as a string, 
+# which we convert to an integer before querying the database for the user:
+@login.user_loader
```
Comment (kenneth0810) [inline app/models.py:110]: There's also a load_user() function in routes.py, but I think it makes more sense to leave it here and remove it from routes.py.

**Review by kenneth0810 — APPROVED** [review]
Looks good!

File: app/models.py (hunk 1 above)
Comment (AnhHoaiNguyen-916) [reply app/models.py:63]: I think it's the database to keep track the metadata of each user account like creating date or things like that, but it seems like we could delete it :)

**Pull request merged**